bindings/java/Cargo.toml

```diff
@@ -74,7 +74,8 @@ services-all = [
   "services-mysql",
   "services-redb",
   "services-redis",
-  "services-rocksdb",
+  # FIXME: rocksdb will lead to "cannot allocate memory in static TLS block" while linking.
```

Comment:
Why do we have this shortcoming?

Reply:
> Why do we have this shortcoming?

Not sure, seems related to static linked librocksdb.